src/kallisto/methods.py

```diff
@@ -145,7 +145,7 @@ def getProximityShells(
 
 
 def getAtomicPartialCharges(
-    at: np.ndarray, coords: np.ndarray, cns: np.ndarray, charge: int
+    at: np.ndarray, coords: np.ndarray, cns: np.ndarray, charge: int, ies: bool
 ):
```

Comment:
Not sure if `getAtomicPartialCharges` is a good name for a function that returns energies. Also, this might be a breaking API change for users of `kallisto.methods`.

Reply:
That's true. I guess it would be better to change this to ``ies=False`` in the definition of ``getAtomicPartialCharges``. 
```python
def getAtomicPartialCharges(at: np.ndarray, coords: np.ndarray, cns: np.ndarray, charge: int, ies=False)
...
````
Then I could return partial charges, the A-matrix, and the X-vector if ``ies=True``, otherwise only the partial charges. For this to work, I need to call the method directly from within the console instead of indirectly within the molecule class. This enables the calculation of the isotropic electrostatic interaction energy inside the console. There is unfortunately still the drawback with the naming since I would return the A-matrix and the X-vector additionally to the partial charges.

Reply:
Good suggestion, this seems to be much clearer. I will encapsulate the eeq part and add a similar method as ``solve`` to the class. Furthermore, I'll add a note that the eeq part in methods will be dumped in the next major release, but I keep it right now to keep the API consistent. Thanks!